fyle_rest_auth/authentication.py

```diff
@@ -64,13 +66,32 @@ def validate_token(access_token_string: str) -> Dict:
 
             api_headers = {'Authorization': '{0}'.format(access_token_string)}
 
-            response = requests.get(my_profile_uri, headers=api_headers)
+            email_unique_key = 'email_' + unique_key_generator[2]
+            user_unique_key = 'user_' + unique_key_generator[2]
 
-            if response.status_code == 200:
-                result = json.loads(response.text)['data']
+            email = cache.get(email_unique_key)
+            user = cache.get(user_unique_key)
+
+            if email is None:
+                if user is None:
```

Comment:
why is this nested if required, it can just be
`if email and user:` right?
also there is no need to mention `is None`

Reply:
`if not (email and user):` works - 
Checking if there is email and user exists in the cache -
if not then make a call to api and store them in cache